docker-compose.yml

```diff
@@ -34,7 +34,7 @@ web:
     - db
     - redis
 celery:
-  build: .
+  image: lore_web
```

Comment:
Question: I'm curious about what this change does.  Did Celery just build everything before, and now it's using an existing image?